magento/magento2
# Resolve [Authorize.net accept.js] "Place Order" button not being disabled when editing billing address

### Description (*)


magento/magento2#23624: Resolve [Authorize.net accept.js] "Place Order" button not being disabled when editing billing address

Reason: Template app/code/Magento/AuthorizenetAcceptjs/view/frontend/web/template/payment/authorizenet-acceptjs.html doesn't have the code to disable Place Order button.

Solution: Need to add the code: `css="disabled: !isPlaceOrderActionAllowed()"` to the template of AuthorizenetAcceptjs Payment.

### Fixed Issues (if relevant)

1. magento/magento2#23624: [Authorize.net accept.js] "Place Order" button not being disabled 

### Manual testing scenarios (*)


1. Configure and setup authorize.net accept js payment method
2. Place non-free item in cart
3. Proceed to checkout.
4. Enter shipping information and proceed to the billing step.
5. Use Authorize.net accept js payment method.
6. Uncheck "My billing and shipping address are the same"
7. Fill out billing information + card details but don't hit the "update" button
8. Results : The place order button at the bottom should be disabled as long as billing information is not updated (Similiar to how the old authorize.net extension, the brain tree extension and the paypal extension work) **=> It prevents the user placing order without Update billing address.**


### Questions or comments


### Contribution checklist (*)
